src/core/public/chrome/ui/header/header_controls_container.scss

```diff
@@ -4,19 +4,21 @@
 */
 
 .headerControl {
-  // stylelint-disable-next-line @osd/stylelint/no_modifying_global_selectors
-  & > .euiHeaderSectionItem {
-    margin: 0 $euiSizeXS;
-  }
+  gap: $euiSizeS;
 
   &.headerDescriptionControl {
-    padding: 0 $euiSizeM;
-    color: $euiTextSubduedColor;
-  }
+    .descriptionHeaderControl {
+      max-width: $euiLegibilityMaxWidth;
```

Comment:
wouldn't does this not look weird if its above a callout that's full width?

Reply:
Will let UX decide.